src/storable.rs

```diff
@@ -70,6 +68,51 @@ pub enum Bound {
     },
 }
 
+impl Bound {
+    /// Returns true if the type is bounded, false otherwise.
+    pub const fn is_bounded(&self) -> bool {
+        matches!(self, Bound::Bounded { .. })
+    }
+
+    /// Returns the maximum size of the type if bounded, panics if unbounded.
+    pub const fn max_size(&self) -> u32 {
+        if let Bound::Bounded { max_size, .. } = self {
+            *max_size
+        } else {
+            panic!("Cannot get max size of unbounded type.");
+        }
+    }
+
+    /// Returns true if the type is fixed in size, false otherwise.
+    /// Panics if unbounded.
+    pub const fn is_fixed_size(&self) -> bool {
+        if let Bound::Bounded { is_fixed_size, .. } = self {
+            *is_fixed_size
+        } else {
+            panic!("Cannot get fixed size of unbounded type.");
```

Comment:
I'd return false here rather than panic.

Reply:
Thank you for the suggestion. I agree that if we decide to remove the other utility methods and keep the interface minimal, returning `false` instead of panicking would make more sense. This approach would be more in line with the principle of least astonishment for the users of the library.

I'll go ahead and update the `is_fixed_size` method to return `false` for unbounded types.